src/App.js

```diff
@@ -19,9 +23,34 @@ export default function App() {
       currentDifficulty = "???";
     }
     setSudokuDifficulty(currentDifficulty);
+
+    if (JSON.parse(localStorage.getItem("currentSudoku")) !== null) {
```

Comment:
It would be sufficient to just write JSON.parse(localStorage.getItem("currentSudoku")) right? 
Because it either evaluates to something which is truthy or null/ undefined which is falsey